dio/src/main.rs

```diff
@@ -0,0 +1,105 @@
+use clap::Parser;
+use diamond_io::{
+    bgg::circuit::serde::SerializablePolyCircuit,
+    io::{obf::obfuscate, params::ObfuscationParams, serde::SerializableObfuscationParams},
+    poly::{
+        Poly, PolyElem, PolyParams,
+        dcrt::{
+            DCRTPoly, DCRTPolyHashSampler, DCRTPolyMatrix, DCRTPolyParams, DCRTPolyTrapdoorSampler,
+            DCRTPolyUniformSampler, FinRingElem,
+        },
+        sampler::{DistType, PolyUniformSampler},
+    },
+    utils::init_tracing,
+};
+use keccak_asm::Keccak256;
+use rand::Rng;
+use serde::Deserialize;
+use std::{
+    fs::File,
+    io::{BufReader, Read},
+    path::PathBuf,
+    sync::Arc,
+};
+use tracing::info;
+
+/// Simple program to obfuscate and evaluate
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+struct Args {
```

Comment:
Note for future updates: the current command implementation can be defined as "bench" subcommand. This implies we might have more subcommands in the future.